DESCRIPTION

```diff
@@ -1,26 +1,38 @@
-Package: ji.rpkg.template
-Title: Your Package Title in Title Case
-Version: 0.0.0.9000
+Package: daedalus.data
+Title: DAEDALUS data
+Version: 0.0.1
 Authors@R: c(
-    person("Pratik", "Gupte", , "pratik.gupte@lshtm.ac.uk", role = c("aut", "cre"),
-           comment = c(ORCID = "0000-0001-5294-7819")),
+    person("Pablo", "Perez-Guzman", , "p.perez-guzman@imperial.ac.uk", role = c("aut", "cre"),
+           comment = c(ORCID = "0000-0002-5277-5196")),
```

Comment:
Could you add Patrick and Rob for initially collecting the data in one place as well?